var/spack/repos/builtin/packages/onednn/package.py

```diff
@@ -85,12 +87,14 @@ class Onednn(CMakePackage):
         values=("ocl", "none"),
         multi=False,
     )
+    variant("acl", default=False, description="Use Arm Compute Library", when="target=aarch64:")
```

Comment:
Any idea what version of onednn added this feature? If not I can try grepping all versions using the script above.

Reply:
I don't know from the top of my head.

Reply:
```suggestion
    variant("acl", default=False, description="Use Arm Compute Library", when="@1.7: target=aarch64:")
```
Looks like 1.7 is the answer.